specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json

```diff
@@ -2459,6 +2459,24 @@
       ],
       "description": "VPN client revoked certificate of virtual network gateway."
     },
+    "RadiusServer": {
```

Comment:
Is this a breaking change as you add new property to the response?

Reply:
No.